src/containers/LoginPage/components/FormLogin.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button';
+import styled from 'styled-components';
+
+
+const LoginFormControl = styled(FormControl)`
+  padding: 10em;
+  position: absolute;
+  margin: auto;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+`
+const LoginButton = styled(Button)`
+  float: left;
+`
+const LoginGithubButton = styled(Button)`
+  margin:auto;
+  display:block;
+`
+
+class FormLogin extends Component {
```

Comment:
```suggestion
class LoginForm extends Component {
```
sửa giúp a tên nếu có thời gian nhé :)